FinancialTransactionsApi/V1/Boundary/Response/MetadataModel.cs

```diff
@@ -0,0 +1,11 @@
+using Newtonsoft.Json;
+
+namespace FinancialTransactionsApi.V1.Boundary.Response
+{
+    public class MetadataModel
+    {
+        [JsonProperty("pagination", NullValueHandling = NullValueHandling.Ignore)]
+
```

Comment:
remove this new line between the property and the attribute pls

Reply:
line has been removed